src/pfs/pfs.proto

```diff
@@ -140,6 +141,17 @@ message Trigger {
   string cron_spec = 6;
 }
 
+message BranchPropagationSpec {
+  // TODO: Picker?
+  Branch branch = 1;
```

Comment:
Yes, a picker would be better here. Not a dealbreaker though.

Reply:
I think I'm going to keep this as is to be consistent with the rest of the `CreateBranch` functionality. Also, this is the proto we use internally, so I think we may use a separate one in the future when we do a V2 of `CreateBranch` that has pickers.